28: human curated tests for generated text

sgol13 opened the pull request.

karol-t-wilk reviewed (commented):
I think it would be good to let the user decide the filepaths for the input/output files. Preferably, the default output file would be added to .gitignore and removed from the index perhaps?

sgol13 commented:
> I think it would be good to let the user decide the filepaths for the input/output files. Preferably, the default output file would be added to .gitignore and removed from the index perhaps?

Good idea, I refactored it a bit:
- Changed a script to a module `BotManualTests`. The module needs to be loaded and then the `run` method can be invoked. Otherwise, arguments could not be passed.
- Created `manual_tests/questions/` and `manual_tests/responses/` directories. The former one is added to git. The  latter one is added to `.gitignore` itself but its content is excluded.
- Default files `questions_1.txt` and `responses_1.html` are added but a user can use the script on other files, as long as they're in proper directories (to not make a mess).
